packages/e2e-tests/specs/adding-blocks.test.js

```diff
@@ -29,6 +29,9 @@ describe( 'adding blocks', () => {
 	}
 
 	it( 'Should insert content using the placeholder and the regular inserter', async () => {
+		// This ensures the editor is loaded in navigation mode.
+		await page.reload();
```

Comment:
Makes me wonder if there's any value in being able to disable it per test case.

Reply:
90% of our tests require it to be disabled. Initially, I tried disabling it per test but I noticed that I was doing it for almost all the tests.

Reply:
I know, I meant having some sort of option to disable it when creating a new post for example. Maybe not worth it...

Reply:
Right! I think it's a good thing to consider. Let's see how this gets affected by #17088 when implemented.